CONTRIBUTING.md

```diff
@@ -324,14 +328,51 @@ Secretless supports using [Go plugins](https://golang.org/pkg/plugin/) to extend
 its functionality. To learn about writing new Secretless plugins and for more
 information on the types of plugins we currently support, visit the [plugin API directory](pkg/secretless/plugin).
 
-### Submodules
+## Submodules
 
 Secretless makes use of some plugins using [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules). 
```

Comment:
I'm not sure `plugins` is the right word here, and it could be confused with our own Secretless plugins - can we change to `third party libraries` or something?